sections/combat.tex

```diff
@@ -186,33 +186,38 @@ \subsection*{\hypertarget{Combatexperience}{Combat Experience}}
 The player is considered to have beaten the Neutral Units by default and gains no rewards from the Combat.
 
 \subsection*{\hypertarget{AIrules}{Player vs AI}}
 These rules apply when playing a \textbf{Solo} game.\index{Solo Mode}
-The AI Combat rules for targeting and attacking are also used when playing a \textbf{Cooperative} Scenario.\par
-Solo Campaigns are played against AI Heroes who use two automated Decks to play the game: the AI Deck, and the Spell Deck.
+The AI Combat rules for unit behaviour are also used when playing a \textbf{Cooperative} Scenario.\par
+Solo Campaigns are played against AI Heroes who use two automated Decks to play the game: the \textbf{AI Deck}, and the \textbf{Spell Deck}.
+Each Campaign Scenario contains instructions for constructing these Decks.\par
 The AI Deck consists of cards that are similar in function to Abilities and Artifacts, but change depending on the game's \hyperlink{Difficulty}{Difficulty}.
+When an \textbf{AI hero} Activates a Unit, draw an AI Card\index{AI Card} and follow its instructions before the Unit moves and/or attacks.
 The Spell Deck should be used when instructed by the cards in the AI Deck.
-During Combat against Neutral enemies or AI Heroes, their Units follow an automatic set of instructions.
+If an AI Hero is instructed to draw a card, they will draw and resolve another card from the AI Deck.\par
+During Combat against Neutral enemies or AI Heroes, their Units follow an automatic set of instructions:
+\par
 \textbf{Initiative rules} work identically.
-When an \textbf{AI hero} Activates a Unit, draw an AI Card\index{AI Card} and follow its instructions as the Unit's Activation.\par
-All AI controlled \includesvg[height=10px]{\svgs/unit_ground.svg} and \includesvg[height=10px]{\svgs/unit_flying.svg} Units prioritize attacking Units of the same tier.
+\includesvg[height=10px]{\svgs/unit_ground.svg} and \includesvg[height=10px]{\svgs/unit_flying.svg} Units prioritize attacking Units of the same tier.
 If this is impossible, they attack the closest Unit, prioritizing the lowest tier one.
 \includesvg[height=10px]{\svgs/unit_ranged.svg} Units prioritize attacking other \includesvg[height=10px]{\svgs/unit_ranged.svg} Units of the same tier, then lower tier, and finally higher tier.
 If there are no \includesvg[height=10px]{\svgs/unit_ranged.svg} Units for them to target they prioritize \includesvg[height=10px]{\svgs/unit_ground.svg} and \includesvg[height=10px]{\svgs/unit_flying.svg} Units in the same tier order.
 If there's more than one valid target, they attack the closest one.\par
-If there's ever a tie between equally valid targets for any Units, the players choose which Unit is attacked.
+If there's ever a tie between equally valid targets for any Units, the players choose which Unit is attacked. Enemy units cannot \hyperlink{Defend}{Defend} unless instructed to.
 
 \textbf{AI Heroes} always start in their Town's Field unless otherwise stated.
```

Comment:
I might be overthinking this, but here's how I see it. Maybe half of this subsection is about actual combat. There could be a separate chapter called "AI Heroes" with this text here starting it. "Player vs AI" subsection would be called "Player vs Neutral Units", which would describe combat only (_During Combat against Neutral enemies [...] Enemy Units cannot Defend unless instructed to._)
The rest of the text (about cards, game modes) would go to AI Heroes, with a hyperlink to "Player vs Neutral Units", explaining that AI Heroes follow the same logic while fighting. WDYT?

Reply:
I'll make a separate PR for this later. It's clear that some re-organization should be done about these.